ERC-20/contracts/LeoToken.sol

```diff
@@ -1,17 +1,32 @@
 //SPDX-License-Identifier: Unlicensed
 pragma solidity ^0.8.0;
+import 'hardhat/console.sol';
```

Comment:
Nie bij, nie jestem do końca zadowolony z projektu bo brakło mi czasu na jego dopieszczenie. Dlatego zostawiłem jeszcze obsługę console.loga bo mam zamiar troche to usprawnić.